core/templates/dev/head/css/oppia.css

```diff
@@ -1027,6 +1030,17 @@ textarea {
   min-width: 126px;
 }
 
+.oppia-library-group-header-active {
```

Comment:
I thought, in a previous comment, we talked about this being an additive class instead of one that replaces .oppia-library-group-header? That would reduce duplication -- see previous comment for more details.